activestorage/test/template/audio_tag_test.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require "test_helper"
+require "database/setup"
+
+class ActiveStorage::AudioTagTest < ActionView::TestCase
```

Comment:
There were tests for `image_tag` with variants and previews, but I'm unsure if those work with audio and videos, so I left them out. I'd happily add them if someone confirms that variants and previews work with audio and video as well.